bindings/python/examples/client/mqtt.py

```diff
@@ -22,8 +24,10 @@
 
 
 def callback(event):
+    """Callback function for the MQTT listener"""
     event_dict = json.loads(event)
     print(event_dict)
+    # pylint: disable=global-statement
```

Comment:
I would allow to use `global` here.
 - makes the example simple
 - otherwise we need to build a stateful callback listener
 - python doesn't allow to pass by reference non-compound types

Reply:
Alright